src/FakeItEasy/Core/CompletedFakeObjectCall.cs

```diff
@@ -15,7 +15,7 @@ public CompletedFakeObjectCall(IInterceptedFakeObjectCall interceptedCall, objec
             this.SequenceNumber = SequenceNumberManager.GetNextSequenceNumber();

```

Comment:
I can't make  a comment any higher, but do we expect the `returnValue` provided in the constructor to survive? Could it be dispensed with?

Reply:
Yes, I'll remove it. Thanks!